Library/Homebrew/cmd/update.sh

```diff
@@ -372,15 +372,20 @@ EOS
   if [[ -n "$HOMEBREW_FORCE_BREWED_CURL" &&
       ! -x "$HOMEBREW_PREFIX/opt/curl/bin/curl" ]]
   then
-    brew install curl
+    ohai "Installing Homebrew cURL"
+    brew install curl || odie "Failed to install cURL"
   fi
 
   if ! git --version &>/dev/null ||
      [[ -n "$HOMEBREW_FORCE_BREWED_GIT" &&
       ! -x "$HOMEBREW_PREFIX/opt/git/bin/git" ]]
   then
     # we cannot install brewed git if homebrew/core is unavailable.
-    [[ -d "$HOMEBREW_LIBRARY/Taps/homebrew/homebrew-core" ]] && brew install git
```

Comment:
I removed this check because here we've got `HOMEBREW_FORCE_BREWED_GIT` set, so Homebrew should fail it it fails install brewed git.

Reply:
Please restore it. It'll try to tap Homebrew/homebrew-core and fail on that which more worse than the `odie` message below.

Reply:
Restored